addons/mail/static/src/scss/res_config_settings_view.scss

```diff
@@ -0,0 +1,4 @@
+.o_settings_color_picker {
```

Comment:
Guidelines: should be names res_config_settings_views.css :)  (as it may contain css for several view types, like other _views files)